src/yaml2candid/Cargo.toml

```diff
@@ -11,3 +11,7 @@ candid_parser = { workspace = true }
 serde = "1"
 serde_yaml = "0.9"
 anyhow = "1"
+num-bigint = { workspace = true }
+
+[dev-dependencies]
+pretty_assertions = "1.4.0"
```

Comment:
Provides a pretty diff in `assert_eq!()` errors.  This makes debugging much easier.